source-browser.html

```diff
@@ -182,6 +193,27 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
       }
 
       $(document).ready(() => {
+        $(window).on('scroll resize load', () => {
+          const top = $(window).scrollTop();
+          const bottom = top + $(window).height();
+          TOPICS.forEach(({ name }) => {
+            const navLinkPosition = $(`#${name}`).position().top;
+            $(`#${name}-nav-link`).text(`${name} ${navLinkPosition > bottom ? '↓' : '↑'}`);
+            $(`#${name}-nav-list`).toggle(top >= navLinkPosition || navLinkPosition >= bottom);		
+          });
+        });
+
+        $('#navbar').append(
+          TOPICS.map(({ name }) =>
+            $('<li class="left">', {id: `${name}-nav-list`}).append(
+              $('<a class="inline-block white center btn:hover">', {
```

Comment:
Can we remove the text underline on these? It's quite odd. Maybe use some of the styles from this page's http://basscss.com/v7/docs/guides/ui/#! Navigation section?